modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderListDisplayContext.java

```diff
@@ -55,8 +62,35 @@ public CommerceOrderListDisplayContext(
 		_keywords = ParamUtil.getString(renderRequest, "keywords");
 	}
 
+	public List<DropdownItem> getBulkActionDropdownItems() {
+		List<DropdownItem> bulkActions = new ArrayList<>();
+
+		PortletDisplay portletDisplay =
+			_commerceOrderRequestHelper.getPortletDisplay();
+
+		ThemeDisplay themeDisplay =
+			_commerceOrderRequestHelper.getThemeDisplay();
+
+		PortletURL portletURL = PortletURLFactoryUtil.create(
+			_commerceOrderRequestHelper.getRequest(), portletDisplay.getId(),
+			themeDisplay.getPlid(), PortletRequest.ACTION_PHASE);
+
+		portletURL.setParameter("redirect", themeDisplay.getURLCurrent());
+		portletURL.setParameter(ActionRequest.ACTION_NAME, "editCommerceOrder");
```

Comment:
Hey @alecsloan @Commando18 I think this should be matching the `mvc.command.name` right?

```suggestion
		portletURL.setParameter(ActionRequest.ACTION_NAME, "/commerce_order/edit_commerce_order");
```

Reply:
Did it, thanks!